### ISSUE TITLE
No output in CI

### ISSUE THREAD

**Issue by TTitscher** [BAMresearch/probeye#60]
Within the CI, the call to `pytest` is wrapped in 
~~~bash
   COVERAGE_REPORT=$(coverage run -m pytest && coverage report -m | tee /dev/stderr)
~~~
which I assume to hide all tests results. This makes it impossible to see the error message or at least which test failed.

**Comment by rozsasarpi** [comment]
If the current way of collecting the coverage is intended to be kept then I assume that it could be solved by printing `COVERAGE_REPORT` to the console. `echo $COVERAGE_REPORT`

**Comment by TTitscher** [comment]
Fixed in 2593f67
